docs/user/centralized_management.rst

```diff
@@ -0,0 +1,515 @@
+containers, and normally these are grouped to correspond to a single
+docker-compose.yaml file.  Each such host needs a HostManager set up,
+though the HostManager runs on the native system and not in a docker
+container.
+
+Config for native system hosts
+------------------------------
+
+Considering the Example Config from :ref:`ocs_site_config_file`, the
+SCF there has 3 hosts defined: ``host-1``, ``host-1-docker``, and
+``host-2``.  We must add a HostManager block to the
+``'agent-instances'`` list in each case.  For example, the ``host-1``
+block would become:
+
+.. code-block:: yaml
+
+    host-1: {
+
+      # Directory for logs.
+      'log-dir': '/simonsobs/log/ocs/',
+
+      # List of additional paths to Agent plugin modules.
+      'agent-paths': [
+        '/simonsobs/ocs/agents/',
+      ],
+
+      # Description of host-1's Agents.
+      # We have two readout devices; they are both Lakeshore 240. But they can
+      # be distinguished, on startup, by a device serial number.
+      # We also have a HostManager.
+
+      'agent-instances': [
+        {'agent-class': 'Lakeshore240Agent',
+         'instance-id': 'thermo1',
+         'arguments': [['--serial-number', 'LSA11AA'],
+                       ['--mode', 'idle']]},
+        {'agent-class': 'Lakeshore240Agent',
+         'instance-id': 'thermo2',
+         'arguments': [['--serial-number', 'LSA22BB'],
+                       ['--mode', 'acq']]},
+        {'agent-class': 'HostManager',
+         'instance-id': 'hm-host-1',
+        },
+      ]
+    }
+
+To test the configuration, you can try to launch the HostManager.  In
+a fully configured system, this will be done through systemd.  But for
+initial setup you can use the ``ocs-local-support`` program.
+
+.. note::
+   When you launch HostManager, it will try to start new processes for
+   each of its managed Agents!  So you should shut down any running
+   instances, and be in a state where it's acceptable to start up new
+   instances.
+
+
+To launch the HostManager agent for the system you're logged into, run::
+
+  $ ocs-local-support start agent --foreground
+
+You can Ctrl-C out of this to kill the agent.  (If you accidentally
+run this without the ``--foreground``, you can try using
+``ocs-local-support stop agent`` to stop it.)
+
+To start using ocsbow to communicate with this HostManager, see
+`Communicating with HostManager Agents`_.  To set the HostManager
+up in systemd (useful especially to have the HostManager and managed
+agents start up when the system boots), see `systemd Control of
+HostManagers`_.
+
+
+Config for docker pseudo-hosts
+------------------------------
+
+Considering the Example Config from :ref:`ocs_site_config_file`, the
+host ``host-1-docker`` describes agents that are launched in
+containers using ``docker-compose``.  After adding a HostManager to
+that pseudo-host the configuration block would look like this:
+
+
+.. code-block:: yaml
+
+    host-1-docker: {
+
```

Comment:
I am not sure why it would require agent-paths.  It doesn't, in my test case.  Agent-paths shouldn't matter unless the HostManager needs to launch an Agent instance (rather than docker-compose something), which it should not do if manage:docker is in all the agent configs.

But ... agreed that this recommendation should be changed so the HM is not defined in the docker host config block.  Then manage:docker won't matter.

Reply:
I think it's because otherwise it won't be able to find the host_manager.py file to run. Trying again I get:
```
$ ocs-local-support start agent --foreground --site-host ocs1-docker
Trying to launch hostmanager agent...
Traceback (most recent call last):
  File "/home/vagrant/.local/bin/ocs-local-support", line 11, in <module>
    load_entry_point('ocs', 'console_scripts', 'ocs-local-support')()
  File "/home/vagrant/git/ocs/ocs/ocsbow.py", line 931, in main_local
    raise OcsbowError('Failed to start manager process: %s' % message)
ocs.ocsbow.OcsbowError: Failed to start manager process: Could not find host_manager.py in the agent_paths!
```

Sounds like it won't be a problem now though. The changes to the docs look good.